packages/camera/camera/test/camera_test.dart

```diff
@@ -1190,6 +1190,133 @@ void main() {
+      // arrange
+      final CameraController cameraController = CameraController(
+          const CameraDescription(
+              name: 'cam',
+              lensDirection: CameraLensDirection.back,
+              sensorOrientation: 90),
+          ResolutionPreset.max);
+
+      await cameraController.initialize();
+      when(CameraPlatform.instance
+              .getVideoStabilizationSupportedModes(mockInitializeCamera))
+          .thenAnswer((_) async => <VideoStabilizationMode>[]);
+
+      // act
+      final Iterable<VideoStabilizationMode> modes =
+          await cameraController.getVideoStabilizationSupportedModes();
+
+      // assert
+      expect(modes, <VideoStabilizationMode>[]);
+    });
+
+    test('getVideoStabilizationSupportedModes() returns off', () async {
+      // arrange
+      final CameraController cameraController = CameraController(
+          const CameraDescription(
+              name: 'cam',
+              lensDirection: CameraLensDirection.back,
+              sensorOrientation: 90),
+          ResolutionPreset.max);
+
+      await cameraController.initialize();
+      when(CameraPlatform.instance
+              .getVideoStabilizationSupportedModes(mockInitializeCamera))
+          .thenAnswer((_) async =>
+              <VideoStabilizationMode>[VideoStabilizationMode.off]);
+
+      // act
+      final Iterable<VideoStabilizationMode> modes =
+          await cameraController.getVideoStabilizationSupportedModes();
+
+      // assert
+      expect(modes, <VideoStabilizationMode>[VideoStabilizationMode.off]);
+    });
+
+    test('getVideoStabilizationSupportedModes() returns all modes', () async {
+      // arrange
+      final CameraController cameraController = CameraController(
+          const CameraDescription(
+              name: 'cam',
+              lensDirection: CameraLensDirection.back,
+              sensorOrientation: 90),
+          ResolutionPreset.max);
+
+      await cameraController.initialize();
+      when(CameraPlatform.instance
+              .getVideoStabilizationSupportedModes(mockInitializeCamera))
+          .thenAnswer((_) async => <VideoStabilizationMode>[
+                VideoStabilizationMode.off,
+                VideoStabilizationMode.on,
+                VideoStabilizationMode.standard,
+                VideoStabilizationMode.cinematic,
+                VideoStabilizationMode.cinematicExtended,
+              ]);
+
+      // act
+      final Iterable<VideoStabilizationMode> modes =
+          await cameraController.getVideoStabilizationSupportedModes();
+
+      // assert
+      expect(modes, <VideoStabilizationMode>[
+        VideoStabilizationMode.off,
+        VideoStabilizationMode.on,
+        VideoStabilizationMode.standard,
+        VideoStabilizationMode.cinematic,
+        VideoStabilizationMode.cinematicExtended,
+      ]);
+    });
+
+    test('setVideoStabilizationMode() calls $CameraPlatform', () async {
```

Comment:
nit: The `$` can be removed from the name of new tests because it has been known to mess up the indexing for some IDEs.

Reply:
I think I didn't miss anything. Thanks for the review!